hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -432,7 +432,9 @@ public static DataStreamSink<Object> clean(Configuration conf, DataStream<Object
   }
 
   public static DataStreamSink<Object> dummySink(DataStream<Object> dataStream) {
-    return dataStream.addSink(Pipelines.DummySink.INSTANCE).name("dummy");
+    return dataStream.addSink(Pipelines.DummySink.INSTANCE)
+        .setParallelism(1)
+        .name("dummy");
```

Comment:
+1, curious about why the append write function has `rebalance` exchange  with this sink, and why they are not chained together.

Reply:
Sorry to jump in...But I think it's more properly to set the parallelism of dummy sink as `FlinkOptions.WRITE_TASKS`, so that dummy sink can be chained with hoodie_write_task, which would reduce resource cost in some cases, e.g., when slot-sharing is disabled.